diy-omd.ino

```diff
@@ -70,15 +129,14 @@ int MODE_ADDR = 6;
 int ON_SEQUENCE_ADDR = 8;
 int OFF_SEQUENCE_ADDR = 12;
 
-/*int OMD_goValue1 = escType == RC ? 135 : 57;
-int OMD_goValue2 = escType == RC ? 135 : 48;
-int OMD_goValue3 = escType == RC ? 135 : 51;*/
-int OMD_goValue1 = escType == RC ? 135 : 27;
-int OMD_goValue2 = escType == RC ? 135 : 23;
-int OMD_goValue3 = escType == RC ? 135 : 23;
+int OMD_goValue1 = OMD_GO_VALUE;
+int OMD_goValue2 = OMD_GO_VALUE;
+int OMD_goValue3 = OMD_GO_VALUE;
```

Comment:
I'd like to clarify the purpose of having 3 separate values instead of just the one, since you've blown the different values away here. We found that due to small hardware and assembly differences, some motors would end up spinning faster or slower. ESC calibration was also a factor until we incorporated a calibration step on every startup. So with a very consistent and precise assembly, it should be less necessary to tweak each one individually.

Reply:
Ah gotcha. I'll split that out again

Reply:
done